login_test.go

```diff
@@ -0,0 +1,25 @@
+package conductoroneapi
+
+import (
+	"context"
+	"fmt"
+	"testing"
+)
+
+func TestLoginFlow(t *testing.T) {
```

Comment:
feels like an odd test case to commit?

it'll run, create dead end devicecodes, but... never finish them?

and... it'll timeout?

feels like the test should be Skip() and return early for now?

Reply:
'twas just a way for me to quickly test the SDK side of this work. Happy to add a skip call in to keep for posterity